api.spec.md

```diff
@@ -1,70 +1,141 @@
-# API V0.1.4
+# API v0.1.4
 
-Point your websocket connection to the hub `ws://ip:3333`
+This is the current API specification for clients that want to connect and send events to the
+Senstate hub.
 
-## Event Types
-Generic Event, sent as json to the hub
-```ts
-interface EventType<T> {
-  event: string,
-  data: T; // one of the following
-}
-```
+- [API v0.1.4](#api-v014)
```

Comment:
I think this line can be removed. no real need to jump 2 lines higher

I thought there was a github-flavoured markdown feature to create the table of content but I couldn't find it.

Reply:
Yeah I know, it's kinda useless to put it but the table of contents feature of the V Code Plugin auto-generates and then updates the table on each save. I will also look for the GitHub Markdown feature.

Let me see if there is an option for it.

Do we even need a toc I wonder? The spec is not that long yet.

Reply:
True, maybe the toc is a bit too much for now anyway^^, lets remove it for now

Reply:
And done. Also found some smaller spelling mistakes and fixed them.